src/discussions/threads/ThreadsView.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ThreadFilterBar from './thread-filter-bar/ThreadFilterBar';
+import Thread, { threadShape } from './thread/Thread';
+
+function ThreadsView({ threads, filterSelfThreads }) {
+  return (
+    <div className="discussion-threads d-flex flex-column card">
```

Comment:
This view is missing the breadcrumbs with the navigation structure present in the mockups.
**Mockups:**
![image](https://user-images.githubusercontent.com/27893385/91783854-c2fbcf80-ebd7-11ea-840f-ad3c9df0c8a8.png)

Reply:
https://edx.github.io/paragon/components/breadcrumbs

Reply:
@giovannicimolin I think it's OK for that to not be there, since the focus here is on the internal UI. The overall UI was meant to be a different task.